### DISCUSSION TITLE
Panasonic cloud

### DISCUSSION THREAD

**Discussion by svanu40** [community/community#33871] [Other Feature Feedback, Questions, & Ideas]
Hi whats wrong with this Panasonic cloud stopped working and got message "This entity is no longer being provided by the panasonic_ac integration. If the entity is no longer in use, delete it in settings."

This a joke? or do anybody have a solution for this? Thanks for help   

Regardes Kaj

**Comment by queenofcorgis** [comment]
Hi @svanu40,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on https://panasoniccustomercommunity.hwy905.com/ca_form.html. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by svanu40** [reply]
Ok thanks for the tip. 👍
